### DISCUSSION TITLE
New models to add

### DISCUSSION THREAD

**Discussion by notprime** [isaaccorley/torchseg#30] [Ideas]
Just a list of new models we should add to torchseg. 
Feel free to comment if you'd like to see any model to be added or if you want to share personal experiences regarding specific archs!

:D

**Comment by notprime** [comment]
- [ ] UperNet
- [ ] SegNext

**Comment by ZappaRoberto** [comment]
Hi, I think you should consider to implement also UNET 3+ (https://arxiv.org/pdf/2004.08790).
In my experience perform bettern than standard Unet or Unet++ :) .

**Reply by JulienMaille** [reply]
You could probably start from this PR https://github.com/qubvel/segmentation_models.pytorch/pull/474

**Comment by GMoncrieff** [comment]
[FeatUp](https://github.com/mhamilton723/FeatUp) looks like a great new decoder (both the JBU and implicit upsamplers). There is an open [issue](https://github.com/qubvel/segmentation_models.pytorch/issues/864) in the smp repo, but nothing has happened since March.

**Reply by isaaccorley** [reply]
I've been using this recently so will definitely add it ASAP!
